src/lib/y2storage/region.rb

```diff
@@ -88,6 +88,15 @@ def cover?(sector)
     #   @param delta [Integer]
     storage_forward :adjust_length
 
+    # @!method adjust_block_size(block_size)
```

Comment:
I would not expose this method in `Y2Storage`. We don't need to call it and I expect we will never do.